demos/browser/ChimeMeetingsWrapper.js

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetingsWrapper {
+
+    // Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
+    // Use the MediaRegion property below in CreateMeeting to select the region
+    // the meeting is hosted in.
+    
+
+    constructor(currentRegion, controlRegion, endpoint){
+        console.log("const controlRegion",controlRegion);
+        this.chime = new AWS.Chime({ region: currentRegion });
+
+        // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com
+        this.chime.endpoint = new AWS.Endpoint(endpoint);
+        this.controlRegion = (!!controlRegion) ? currentRegion : controlRegion;
+        console.log("Const this.controlRegion",this.controlRegion);
+        this.chimeMeetings = new AWS.ChimeSDKMeetings({ region: this.controlRegion});
+    }
+
+    getChimeInstance(glabal){
```

Comment:
1. `global`
2. I wouldn't expect this to be a parameter to be present in each method, I'd expect it to be present in the constructor since we are attempting to abstract away this concept from the caller.

Reply:
removed